api/src/test/scala/uk/ac/wellcome/platform/api/models/DisplayItemTest.scala

```diff
@@ -29,7 +17,7 @@ class DisplayItemTest extends FunSpec with Matchers {
     Location(
       locationType = locationType,
       url = thumbnailUrl,
-      license = license
+      license = License_CCBY
```

Comment:
I think this is my bad – but while we’re here, can we scrub the `license{Type, Label, Url}` definitions on the lines above?

Reply:
Not sure what you mean... Do you want to move the license json string building out?